models/linkedin.yml

```diff
@@ -533,5 +533,350 @@ models:
       - name: external_website_conversions
         description: The actions taken on your website that you've defined as valuable to your business after clicking on LinkedIn ads.
       
+      - name: one_click_leads
+        description: Leads submitted after clicking on LinkedIn ads.
+
+  - name: linkedin_ads__monthly_campaign_country_report
+    description: Each record in this table represents the monthly performance of a campaign at the country level.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - source_relation
+            - date_month
+            - country_name
```

Comment:
I'm seeing some `dbt test` failures because there can evidently be multiple records in the `geo` table per `country_name`. Like Norway
![image](https://github.com/user-attachments/assets/0b0f6911-9641-42b8-887d-3614e3b06bfb)

I wonder if we should drop the `member_country` ID, or use that for the uniqueness test. What do you think? I presume users would prefer to have the grain truly be at the country rather than this ID 🤔

Reply:
Region passes but I wonder if we would want to apply the same to `member_region`